cmd/fortifyExecuteScan.go

```diff
@@ -465,9 +471,12 @@ func getSpotIssueCount(config fortifyExecuteScanOptions, sys fortify.System, spo
 		}
 		flagOutput := ""
 
-		if ((total <= config.SpotCheckMinimum || config.SpotCheckMinimum < 0) && audited != total) || (total > config.SpotCheckMinimum && audited < config.SpotCheckMinimum) {
-			currentDelta := config.SpotCheckMinimum - audited
-			if config.SpotCheckMinimum < 0 || config.SpotCheckMinimum > total {
+		minSpotChecksPerCategory := getMinSpotChecksPerCategory(config, total)
+		log.Entry().Debugf("Minimum spot checks for group %v is %v with audit count %v", group, minSpotChecksPerCategory, audited)
+
+		if ((total <= minSpotChecksPerCategory || minSpotChecksPerCategory < 0) && audited != total) || (total > minSpotChecksPerCategory && audited < minSpotChecksPerCategory) {
```

Comment:
This logic is impenetrable, but given that
a) it's close to what we had before and
b) there is a good amount of tests
I vote to keep it for now.